gatorgrade/output/src/output_functions.py

```diff
@@ -0,0 +1,57 @@
+"""
+This class is used for storing the main functions requested from the Github.
+
+Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output,
+and colorization of text.
+"""
+from colorama import init, Fore, Style
+
+init()
+
+
+def sort_checks_by_result(results):
+    """
+    Process results and determine if the check passed or failed.
+
+    Argument `results` is: list[(check_result_element1, check_result_element2),(...)]]

```

Comment:
If you are going to use this docstring format, can you use the way we wrote it in the previous practical? ex. Args results: explanation